.travis.yml

```diff
@@ -8,6 +8,11 @@ env:
   - ARCH=posix CONF=dbg
   - ARCH=posix CONF=opt
   - ARCH=posix CONF=lto
+before_install:
+  - [ X"$TRAVIS_OS_NAME" = X"osx" ] && brew install erlang
```

Comment:
bash nitpicks: `[ x$something = xtext ]` idiom is useful when there is no quotes. If you have quotes, there is no need to add `x` or anything before a string.